src/main/resources/templates/cypher/list.html

```diff
@@ -35,6 +36,9 @@ <h2>Šifry</h2>
                 <td th:text="${cypher.codeword}">codove slovo</td>
                 <td th:text="${cypher.trapCodeword}">Past</td>
                 <td th:text="${#strings.abbreviate(cypher.bonusContent,50)}">Bonus</td>
+                <td>
+                    <th:block th:unless="${cypher.organizers == null}" th:each="o : ${cypher.organizers}" th:text="${o.username} + ' '">Organizator</th:block>
```

Comment:
Tady by bylo supr, kdyby se to odělovalo čárkama. Ale nevím jestli má Thymeleaf nějakou implode nebo join funkci pro seznam stringů.